src/application/editor.py

```diff
@@ -27,6 +29,10 @@ class ThoughtBox(MenuNav):
     def __init__(self):
         # Create notes directory
         os.makedirs(NOTES_DIR, exist_ok=True)
+        # If welcome page isn't present, create it.
+        if not os.path.isfile(NOTES_DIR + "/" + WELCOME_PAGE):
+            copyfile(ASSETS_DIR + WELCOME_PAGE, NOTES_DIR + "/" + WELCOME_PAGE)
```

Comment:
Is string joining preferable to `os.path.join` in this case?

Reply:
I'm open to it, want to suggest a change using the button?
![image](https://user-images.githubusercontent.com/21180929/125820228-c82e124d-f0f2-48f1-a10f-c9eed9bf23e2.png)